Triangulation_2/include/CGAL/Weighted_point.h

```diff
@@ -25,13 +25,11 @@
 #include <CGAL/license/Triangulation_2.h>
 
 // Below is silenced until Surface_mesher is cleaned from Weighted_point.h
-/*
-#define CGAL_DEPRECATED_HEADER "<CGAL/Weighted_point.h>"
-#define CGAL_DEPRECATED_MESSAGE_DETAILS \
-   "Weighted points are now part of the concept Kernel. One should therefore "\
-   "use `CGAL::Weighted_point_2<K>` and `CGAL::Weighted_point_3<K>`."
-#include <CGAL/internal/deprecation_warning.h>
-*/
+// #define CGAL_DEPRECATED_HEADER "<CGAL/Weighted_point.h>"
+// #define CGAL_DEPRECATED_MESSAGE_DETAILS \
+//   "Weighted points are now part of the concept Kernel. One should therefore "\
+//   "use `CGAL::Weighted_point_2<K>` and `CGAL::Weighted_point_3<K>`."
+// #include <CGAL/internal/deprecation_warning.h>
 
```

Comment:
That is now fixed in https://github.com/CGAL/cgal/pull/2247/commits/bad0922002ac0813ab5fa643813e227abc97a449.